### DISCUSSION TITLE
Are react-native-reanimated transitions supported on web?

### DISCUSSION THREAD

**Discussion by EliotKhachi** [software-mansion/react-native-reanimated#4072] [Q&A]
**Is there web support for react-native-reanimated transitions?**

I'm trying to implement a transition to display a small drop-down when pressing a button, but the transition isn't taking effect. I'm getting the following warning when I invoke the `animateNextTransition()` of my transitioning View:

`Reanimated: animateNextTransition is unimplemented on current platform`

I've tested the same thing on iOS and the transition works as expected. In web, I've already tested a basic `useShareValue()` and `useAnimatedStyle()` translation animation and it works as expected; so, I'm assuming some react-native-reanimated features work and some don't on web. I'm beginning to look at alternative animation packages such as React Spring, but I would prefer to stick with React Native packages if possible.

Thanks,

Eliot

Screenshots of code for web:
![Screenshot from 2023-02-19 11-56-20](https://user-images.githubusercontent.com/68179450/219971954-375cdda3-eed7-41a8-b65d-f06ede173fd8.png)
![Screenshot from 2023-02-19 11-56-40](https://user-images.githubusercontent.com/68179450/219971955-a24c6960-9104-4032-be0e-e8b0874ef09f.png)

**Comment by voody2506** [comment]
Intersting too
